src/NOTAM/NOTAMGlue.cpp

```
@@ -0,0 +1,1518 @@
+    }
+    listener_notify.SendNotification();
+  };
+
+  if (start_queued_refresh) {
+    if (load_committed)
+      apply_committed_load();
+    else
+      Cancel();
+
+    LogFmt("NOTAM: Starting queued forced refresh");
+    load_task.Start(LoadNOTAMsInternal(queued_location, true),
```

Cppcheck (2.20.0)

[error] Unhandled exception thrown in function declared not to throw exceptions.

(throwInNoexceptFunction)